doc/periodictask/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _periodic_tasks:
+
+Periodic Tasks
+==============
+
+.. index:: periodic task, recurring task
+
+Starting with version 2.23, privacyIDEA comes with the ability to define periodically recurring tasks
+in the Web UI. Currently, the primary purpose of such tasks is to periodically collect statistical information,
```

Comment:
I would not say that we currently can only do this and that! Feels a bit small.
Also, we would need to change this text, when new modules appear.

"The pupose of such tasks is to periodically execute certain tasks automatically. Using Task Modules the administrator can define which tasks should be executed when."